src/components/common/forms/SignUpForm.tsx

```diff
@@ -46,8 +46,8 @@ const SignUpForm = () => {
       const response = await signUpRequest({
         user: data,
       });
-      const { user } = response;
-      dispatch(setUsername(user.name));
+      const user = response;
+      dispatch(setUser(user));
```

Comment:
`const user = await signUpRequest({
        user: data,
      });`
      instead of 
      `const user = response;`

Reply:
Love it